script/depots/request_depot.lua

```diff
@@ -404,8 +404,8 @@ function request_depot:update_circuit_writer()
   local circuit_condition = behavior.connect_to_logistic_network and behavior.logistic_condition or behavior.circuit_condition
   if circuit_condition then
     local condition = circuit_condition.condition
-    if condition.comparator == "=" then
-      local first_signal = condition.first_signal
+    local first_signal = condition.first_signal
+    --[[ if condition.comparator == "=" then      
```

Comment:
Why are you commenting this out?

Reply:
It is not documented and when I've found this code, I thought it is not an obvious behavior.

Reply:
It is documented int he in-game tips and tricks:
https://github.com/Klonan/Transport_Drones/blob/master/locale/en/transport_drones.cfg#L71

Reply:
My bad, I'll make another PR when I have the time.